ge.h

```diff
@@ -131,12 +147,20 @@ struct ge {
     /* Registers for special conditions and exceptions */
     uint8_t rFI[7];
 
+    /* 7 bits */
+    uint8_t ffFI;
```

Comment:
there are a few registers that have a non 8/16 number of bits.. what do you think of having some tyedefs like `geuint7_t` `geuint9_t` etc (or similar naming)? even not in the context of this PR?

/cc @danielinux

Reply:
I think it's a good idea.  If we really want to stretch this we may also include if the type refers to a Flip Flop, a register, a knot, or a signal. Even though I think prefixes are already sufficient for that.

Reply:
oh great! i think FFs / registers / knot is a bit too extra mile, unless we need some more complex operations and we want to turn those into structs!